pkg/api/v1beta3/dynakube/feature_flags.go

```diff
@@ -247,6 +251,36 @@ func (dk *DynaKube) FeatureMaxFailedCsiMountAttempts() int {
 	return maxCsiMountAttemptsValue
 }
 
+func (dk *DynaKube) FeatureMaxCSIRetryTimeout() time.Duration {
+	maxCsiMountTimeoutValue := dk.getFeatureFlagRaw(AnnotationFeatureMaxCsiMountTimeout)
+
+	duration, err := time.ParseDuration(maxCsiMountTimeoutValue)
+	if err != nil || duration < 0 {
+		duration, _ = time.ParseDuration(DefaultMaxCsiMountTimeout)
+	}
+
+	return duration
+}
+
+// MountAttemptsToTimeout converts the (old) number of csi mount attempts into a time.Duration string.
+// The converted value is based on the exponential backoff's algorithm.
+// The output is string because it's main purpose is to convert the value of an annotation to another annotation.
+func MountAttemptsToTimeout(maxAttempts int) string {
+	var power float64 = 2
+
+	var baseDelay = 1 * time.Second / 2
```

Comment:
```suggestion
	var baseDelay = 500 * time.Millisecond
```

Reply:
or `time.Second/2` 😛

Reply:
metally easy to read, since you don't need to calculate it.

Reply:
IMO "half of a second" is easier to read than 500 Millisecond (my monkey brain doesn't like milliseconds 🐵 )